platformcomponents/desktop/slider.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slider": {
+        "comment": "",
+        "figma": "https://www.figma.com/file/H19CutixoN8SJGO5EXwcc8/Components---MacOS?node-id=3153%3A1415",
+        "track": {
+            "fill": "@theme-control-active-normal",
+            "empty": "@theme-outline-button-normal"
+        },
+        "knob": {
```

Comment:
in the figma, this is called 'mover'. Can we call it this way to keep consistency?